### PR TITLE
feat(Bluesky): add Bluesky presence

### PR DESCRIPTION
**Pull request by oriionn**
## Description 
Hello !
I've added a richpresence for Bluesky
PS: In my commit there was a small problem and put diffs to some files but I have not touched personally.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>

Home

Post

View Profile

</details>

### CODE REVIEW

File: websites/#/ッツ Ebook Reader/metadata.json
Comment (theusaf) [inline websites/#/ッツ Ebook Reader/metadata.json:1]: It is likely that you committed these files after running the formatter. You can try:
* merging the latest main branch into your branch
* reverting the changes to these files (using git checkout, manually resetting them, or other methods)

File: websites/B/Bluesky/presence.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/uBFoZMP.jpg",
+		},
+		path = document.location.pathname.split("/"),
+		showUsername = await presence.getSetting<boolean>("show_profile"),
+		showPost = await presence.getSetting<boolean>("show_post"),
+		showFeed = await presence.getSetting<boolean>("show_feed");
```
Comment (theusaf) [inline websites/B/Bluesky/presence.ts:12]: combine these calls using `Promise.all`

File: websites/B/Bluesky/presence.ts [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+	switch (path[1]) {
+		case "profile":
+			switch (path[3]) {
+				case "feed":
+					presenceData.details = "Viewing a feed...";
+					if (showFeed) {
+						presenceData.state = document.querySelector(
+							"a.css-1jxf684.r-1loqt21:not(.r-13awgt0)"
```
Comment (theusaf) [inline websites/B/Bluesky/presence.ts:21]: If possible, try to use selectors that don't use generated class names.

File: websites/B/Bluesky/presence.ts [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+		case "feeds":
+			presenceData.details = "Browsing...";
+			presenceData.state = "Feeds";
+			break;
+		case "search":
+			presenceData.details = "Searching...";
+			break;
+		case "notifications":
+			presenceData.details = "Browsing...";
+			presenceData.state = "Notifications";
```
Comment (theusaf) [inline websites/B/Bluesky/presence.ts:66]: Preferably, avoid using `...` in the strings.

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/B/Bluesky/presence.ts (hunk 2 above)
Comment (oriionn) [reply websites/B/Bluesky/presence.ts:21]: image

It is not possible, there are only class for the name.

Comment (theusaf) [reply websites/B/Bluesky/presence.ts:21]: You could try things like `div div > ... > a` (or similar). If that doesn't work, just know that the presence may break or need updates often.

**Comment by oriionn** [comment]
Why the check don't work

File: websites/X/X.com/metadata.json
Comment (theusaf) [inline websites/X/X.com/metadata.json:1]: This file also needs to be reverted.

**Review by theusaf — APPROVED** [review]

**Pull request merged**